x-pack/plugins/apm/public/components/app/RumDashboard/CoreVitals/PaletteLegends.tsx

```diff
@@ -36,10 +55,13 @@ export function PaletteLegends({
   onItemHover,
   thresholds,
 }: Props) {
+  const [darkMode] = useUiSetting$<boolean>('theme:darkMode');
+
   const palette = euiPaletteForStatus(3);
+  const labels = [GOOD_LABEL, NEEDS_IMPROVEMENT_LABEL, POOR_LABEL];
 
   return (
-    <EuiFlexGroup responsive={false}>
+    <EuiFlexGroup responsive={false} gutterSize={'s'}>
```

Comment:
```suggestion
    <EuiFlexGroup responsive={false} gutterSize="s">
```

I'm kind of surprised prettier doesn't already do this and I don't leaving comments like this but it seems more readable to me when bare strings use this syntax.

You don't need to change this, I just wish prettier would force everyone to use one syntax or the other.